### PR TITLE
Brought back the Windows setup deployment via Travis CI

### LINKED ISSUE #8153: AppVeyor Windows build significantly slower than Mono build running on Microsoft Windows
My numbers on Win7x64, i5-2500 CPU, RA shellmap:

OSX release 20150424: avg ~4.0-4.5 Tick / ~2.5-3.0 Render
Win release 20150424: avg ~5.5-6.0 Tick / ~4.0-4.5 Render

So the mono build is at least roughly 30-50% faster with identical compile settings (Debug mode, no optimization).

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/14535367-appveyor-windows-build-significantly-slower-than-mono-build-running-on-microsoft-windows?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Closes #8153
- Travis CI will do the final Windows deployment so every official release CLI get's compiled by the same compiler using the same flags to avoid possible desyncs.
- We switch back the legacy Travis CI non-Docker environment, because we need sudo to install NSIS. ~~The nuget connectivity problems should be gone now thanks to caching.~~ Caching is not supported on the non-container based infrastructure. :(
- I decided to still keep the AppVeyor Setup creation so we keep our nightly builds and CI checks for the NSIS scripts which tend to bit rot faster, because everything has to be listed and the syntax is error prone.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +48,7 @@ notifications:
     skip_join: true
 
 before_deploy:
+ - apt-get download nsis nsis-common
```
Comment (obrakmann) [inline .travis.yml]: You can add the markdown package back again as well, and remove the download of the zip in `package-all.sh`

**Comment by obrakmann** [comment]
A windows package compiled with mono4 [is here](https://seacloud.cc/d/5d1adbebce/files/?p=/openra/OpenRA-mono4-20150520.exe)

**Comment by Mailaender** [comment]
We are still on Mono 3.12 here.

**Comment by Mailaender** [comment]
Solved the `markdown` not working on @openSUSE by https://build.opensuse.org/request/show/308209 and updated the scripts as advised for a more complete revert of #7607.

**Comment by obrakmann** [comment]
There's hoping that travis have improved their network and we won't suffer the timeouts again.

:+1:

**Comment by penev92** [comment]
Here's a :+1: with my limited understanding.

**Comment by Mailaender** [comment]
This is again breaking constantly. I filed https://github.com/travis-ci/travis-ci/issues/3940 for assistance.

**Pull request merged**